docs/add-ins/addin-tutorial.md

```diff
@@ -1049,6 +1155,21 @@ Now that the UI is implemented, let's add the code behind it. Create a file name
 })();
 ```
 
-Save all of your changes and run `npm start` if the server isn't already running. Open Outlook and compose a new message. When you click on the **Insert Gist** button, you should see a task pane open on the right-hand side. When you select a gist and click **Insert**, the gist should get inserted into the body.
+### Test the button
+
+Save all of your changes and run `npm start` via the command prompt, if the server isn't already running. Then complete the following steps to test the **Insert gist** button. 
```

Comment:
Don't use "via", say "from the command prompt"

Reply:
Good suggestion; I will update.